cli/readme.py

```diff
@@ -483,7 +483,7 @@ def write_job_using_registry_components_workflow(job):
     # Duplicate name in working directory during checkout
     # https://github.com/actions/checkout/issues/739
     workflow_yaml = f"""{READONLY_HEADER}
-name: cli-{hyphenated}-registry
+name: registry-cli-{hyphenated}
```

Comment:
Any reason of breaking the protocol for defining the workflow names. 
For SDK, the workflow name starts with sdk-
For CLI, the workflow name starts with cli-